api/ImageData.json

```diff
@@ -161,10 +161,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "14"
+              "version_added": "3.5"
```

Comment:
This and the following features should probably match the parent feature if that changes to match `createImageData`.